docs/guides/migrations.md

```diff
@@ -7,6 +7,15 @@ kb_sync_latest_only
 
 # Migrations
 
+## 4.1 to 4.2
+
+A better handling for cookie `SameSite` and `secure` settings was implemented with new defaults to `SameSite=Strict` and `secure`.
+This can still be overridden when calling `cookies.services` `put` method with explicitly set values.
+Additionally if the PWA is run in an iframe all cookies are set with `SameSite=None` and if run with `http` (should only be in development environments) `secure` is not set.
```

Comment:
May here (or elsewhere) we should mention that running the pwa in an iframe with http is not working properly and will lead to cookie errors.

Reply:
This we need to discuss.